plugins/modules/rabbitmq_user_limits.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2013, Chatham Financial <oss@chathamfinancial.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: rabbitmq_user_limits
+short_description: Manage RabbitMQ user limits
+description:
+  - Manage the state of user limits in RabbitMQ
+author: Aitor Pazos (@aitorpazos)
+options:
+  user:
+    description:
+      - Name of user to manage limits for
```

Comment:
```suggestion
      - Name of user to manage limits for.
```
Must be there. Please fix for the other options where it's missed.

Reply:
Fixed